src/components/checkbox/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { memo } from 'react';
+
+import { RequiredKeys } from '@typings/index';
+
+type InputProps = RequiredKeys<
+  React.DetailedHTMLProps<
+    React.InputHTMLAttributes<HTMLInputElement>,
+    HTMLInputElement
+  >,
+  'id'
+>;
+
+const Checkbox: React.FC<InputProps> = ({ ...props }) => (
```

Comment:
Get the `className` from the spread and pass it to the parent class (via `twMerge`) so that styles can be overridden from the outside